.devcontainer/setup.bash

```
@@ -0,0 +1,6 @@
+rm -r Dockerfile .gitpod.yml .github
+echo -e "# Snakemake-Tutorial\n\nYour gitpod workspace for the snakemake-tutorial has been initialized. Now you can start with the [basic tutorial](https://snakemake.readthedocs.io/en/stable/tutorial/basics.html)." > README.md
+echo \"*\" > .gitignore
+conda init
+source ~/.bashrc
+echo 'conda activate snakemake-tutorial' >> ~/.bashrc"
```

Shellcheck (0.10.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)